libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/MediaUtils.java

```diff
@@ -71,6 +71,13 @@ public static boolean isVideo(String url) {
                 url.endsWith(".3g2");
     }
 
+    public static boolean isAudio(String url) {
+        if (url == null) {
+            return false;
+        }
+        return url.endsWith(".mp3") || url.endsWith(".ogg") || url.endsWith(".wav");
```

Comment:
This will catch the most prevalent audio file formats, but might be worth adding .wma, .aiff, .aif, .aac, and .m4a.